_posts/2016-09-12-Microservices-Dashboard-1.0.1.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+authors: [andreas_evers]
+title: 'Microservices Dashboard'
+image: /img/microservices-dashboard.png
+tags: [Microservices, Dashboard, Spring, Spring Boot, RxJava, Pact, Hypermedia, Hateoas]
+category: Microservices
+comments: true
+---
+
+<img class="center-block" alt="Architecture" src="/img/microservices-dashboard/screenshot.png" width="70%">
+
+
+So you’ve jumped on the hype train, built a bunch of microservices, and got your first releases under your belt. 
+Now what?
+Our experiences taught us this is the easy part.
+With the newly obtained microservices freedom, teams easily plunge into a world of cowboys and unicorns.
+The big ball of mud is just around the corner.
+Panic, mayhem and chaos loom over the organisation, waiting for everything to spin out of control.
+Especially for any enterprise not residing in silicon valley, maintaining some sort of governance and compliancy is essential.
+
+What does a microservice architecture mean not just for the developer, but also for analysts and managers?
```

Comment:
Would add an extra comma and _developers_ (plural):

> What does a microservice architecture mean**,** not just for the developer**s**, but also for analysts and managers?

Or maybe rephrase the sentence a bit since it looks a bit odd...
> What does a microservice architecture mean...? For the developers...? And for the analysts, managers, ...?